telegram/ext/jobqueue.py

```diff
@@ -51,10 +49,6 @@ def __init__(self, bot, prevent_autostart=False):
         """:type: float"""
         self._running = False
 
-        if not prevent_autostart:
-            self.logger.debug('Auto-starting %s', self.__class__.__name__)
-            self.start()
-
     def put(self, job, next_t=None):
         """Queue a new job. If the JobQueue is not running, it will be started.
```

Comment:
This docstring is wrong (and was wrong before your PR). The job queue is never started in this method. Please remove that sentence.